metricflow/cli/main.py

```diff
@@ -658,21 +657,7 @@ def validate_configs(
     if not show_all:
         print("(To see warnings and future-errors, run again with flag `--show-all`)")
 
-    # Skip linting validation for dbt cloud
-    if cfg.dbt_cloud_configs is not None:
-        lint_results = ModelValidationResults()
-    else:
-        # Lint Validation
-        lint_spinner = Halo(text="Checking for YAML format issues", spinner="dots")
-        lint_spinner.start()
-
-        lint_results = ConfigLinter().lint_dir(path_to_models(handler=cfg.config))
-        if not lint_results.has_blocking_issues:
-            lint_spinner.succeed(f"🎉 Successfully linted config YAML files ({lint_results.summary()})")
-        else:
-            lint_spinner.fail(f"Breaking issues found in config YAML files ({lint_results.summary()})")
-            _print_issues(lint_results, show_non_blocking=show_all, verbose=verbose_issues)
-            return
+    lint_results = ModelValidationResults()
```

Comment:
Can we remove this altogether? Looks like it's only used in the removed block, plus as an input into a multi-result merge, so we should be able to delete it from that array as well.

Reply:
Removed it!